### DISCUSSION TITLE
Filtering with schema first approach

### DISCUSSION THREAD

**Discussion by joao-figueira** [ChilliCream/hotchocolate#3773] [Q&A]
Is there any workaround to use filtering with schema first approach? I saw on the documentation that it's not currently supported, but is there any "ugly" way to do it? 

In my case, I have no other option than define the schema using SDL (schema-first) but I really want to use the filtering feature.

I tried to manually include the filters in the schema and add the ".AddFiltering()" to the GraphQL server, but it doesn't filter anything. Any ideas? :)

Thanks guys

![image](https://user-images.githubusercontent.com/56129502/120318686-0b1b6000-c2d8-11eb-9ef6-26d8f64e2f1a.png)

**Comment by michaelstaib** [comment]
Support is coming with v12
